portfolio-project-matching-app/components/FilterButtons.js

```diff
@@ -39,8 +43,8 @@ const FilterButtons = ({ category, choices, onClick }) => {
             <div>
                 {selectedChoices.map((choice) => {
                     return (
-                        <div className='py-1 pr-2'>
-                            <Button key={choice.id} type='btnWarning' text={choice.name} onClick={() => removeFilter(choice)} />
+                        <div key={choice.id} className='py-1 pr-2'>
```

Comment:
I didn't make either of these changes, but had merged with Charlie's changes per Josh's suggestion on Slack about a week ago.

Reply:
I'm second guessing if I told you the correct way know. Maybe I don't know the correct way.

Reply:
Shouldn't be too big of a problem how the changes got to this branch, as long as we agree that the changes are ok.